CTP/README.md

```diff
@@ -122,40 +124,63 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
   
 - **SHELL**
   - Prepare
-	* Prepare at least two accounts (e.g., one account ``controller`` as controller, another account ``shell_instance1`` as test instance)
-	* Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``shell_instance1``
-	
-	  ```
+        * For local test
+	   * Prepare one account to use for controller and test instance
+	   * CTP and test scenario should be checked out on it, and configure JAVA_HOME
+	   ```
+	   JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+	   ```
+	   * **Example** ``shell.conf`` for a single instance, scenario and test build:
+	   ```
+	   default.cubrid.<property> //configure parameters for cubrid.conf

```

Comment:
Change the comment format and give a value to the property:
as is:
default.cubrid.<property> //configure parameters for cubrid.conf

to be:
#configure parameters for cubrid.conf
default.cubrid.\<property\>=\<property value\>